docs/mobile-apps/live-testing/live-mobile-app-testing.md

```diff
@@ -198,6 +198,16 @@ There are three ways to test Apple Pay with Sauce Labs:
 * Using real private devices with an Apple Pay Sandbox Testing account
 * Using real private devices with a real production account and real credit cards
 
+### Requirements
+- You need to use [Private devices](#apple-pay-on-real-private-devices)
+- [Resigning](#disable-re-signing) needs to be disabled
```

Comment:
Resigning is the capability for automated. this is livetesting so lets rename this to Instrumentation so it is matching with the UI otherwise it will be super confusing